deploy/kustomize/deploy_kustomize.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+set -e
+# Setup Environment Variables
+export TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+export APP_REGISTRY="${APP_REGISTRY:-kubevirt-hyperconverged}"
+export CSC_SOURCE="${CSC_SOURCE:-hco-catalogsource-config}"
+export CS_SOURCE="${CS_SOURCE:-hco-catalogsource}"
+export PACKAGE="${PACKAGE:-kubevirt-hyperconverged}"
+export MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
+export HCO_VERSION="${HCO_VERSION:-1.0.0}"
+export HCO_CHANNEL="${HCO_CHANNEL:-1.0.0}"
+export PRIVATE_REPO="${PRIVATE_REPO:-false}"
+export QUAY_USERNAME="${QUAY_USERNAME:-}"
+export QUAY_PASSWORD="${QUAY_PASSWORD:-}"
+export QUAY_TOKEN=""
+export AUTH_TOKEN=""
+export CREATE_NAMESPACE="${CREATE_NAMESPACE:-}"
+
+BASE_DIR=base
+if [ "$1" = 'marketplace' ]; then
+  OVERLAY_DIR=overlays/marketplace
+elif [ "$1" = 'image_registry' ]; then
```

Comment:
What's image_registry in that sense?

Reply:
- `marketplace` option is deployment using CatalogSourceConfig, which creates the CatalogSource from the OperatorSource. This is equivalent to `deploy_marketplace.sh`.
- `image_registry` option is deployment using CatalogSource, where you specify the bundle image of the operator directly. This is equivalent to `deploy_imageregistry.sh`.